docs/building_updating_flank.md

```diff
@@ -0,0 +1,23 @@
+# Building and Updating Flank
+
+Ensure that all steps taken for contributing and building Flank have been followed, which are found [here](../README.md)
+
+## Building an updated flank
+To build an updated version of flank from source simply run (This assumes you are in the root Flank directory)
+
+> ./gradlew clean build
+> ./gradlew updateFlank
```

Comment:
nit: Recently I've added the task `flankFullRun` and what it does:
1) clean
2) check (depends on detekt)
3) run integration tests
4) invoke `update_flank.sh` script (like `updateFlank` task)

just FYI

Reply:
I'll update it for that, it's definitely better choice.